patches/wlan-ap/0007-ipq40xx-TP-Link-AP2220-support.patch

```diff
@@ -233,6 +233,7 @@ index 0000000000..1746f51d54
 +			status = "okay";
 +			switch_mac_mode = <0x0>; /* mac mode for RGMII RMII */
 +			switch_initvlas = <0x0007c 0x54>; /* port0 status */
++			witch_initvlas = <0x0007c 0x54>; /* port0 status */
```

Comment:
this must have snuck in, did you do some merge commit with git causing this ?

Reply:
Yes I did, I was working on staging-osync2.0. So I checked out to pending branch, and did merge my previous working branch on staging-osyn2.0 to local branch on pending. Because of that it might stuck in.

Reply:
This is for editing the core code. we are building our own platform code however. the current platform code i wrote follows the kernel coding style and is c99 compliant. This is what opensource folks would expect to find and aligns with the entire openwrt code base.